pkg/reconciler/workload/defaultlocation/defaultlocation_controller.go

```diff
@@ -324,20 +300,5 @@ func (c *controller) process(ctx context.Context, key string) error {
 		return err
 	}
 
-	// patch the apiexport, so we do not create the location/apibinding again even if it is deleted.
```

Comment:
Do we need to keep a similar logic to avoid creating the location/apibinding again even if it is deleted ?

Reply:
Can we avoid this annotation at all ?

we could also change a bit the logic and do the following:
- Create / delete the local APBinding systematically according to the presence / absence of the local APIExport
- Create the default location if no location already exists.